client/src/components/relatedProducts/Modal.jsx

```diff
@@ -9,38 +10,69 @@ const Modal = (props) => {
   if (comparedProduct.product !== undefined && comparedProduct.product.features !== undefined) {
     // eslint-disable-next-line no-plusplus
     for (let i = 0; i < currentProduct.features.length; i++) {
-      if (currentProduct.features[i].value === true) {
-        currentProduct.features[i].value = <FontAwesomeIcon icon={'faCheck'} />;
-      } else if (currentProduct.features[i].value === false) {
-        currentProduct.features[i].value = '';
-      }
       features[currentProduct.features[i].feature] = {
         currentProduct: currentProduct.features[i].value,
       };
     }
     // eslint-disable-next-line no-plusplus
     for (let j = 0; j < comparedProduct.product.features.length; j++) {
-      if (comparedProduct.product.features[j].value === true) {
-        comparedProduct.product.features[j].value = <FontAwesomeIcon icon={'faCheck'} />;
-      } else if (comparedProduct.product.features[j].value === false) {
-        comparedProduct.product.features[j].value = '';
+      if (!features[comparedProduct.product.features[j].feature]) {
```

Comment:
assigning `comparedProduct.product.features[j].feature` to a variable and refer to that variable afterwards may help? feel like needing to write it out every time for me is a pain 🤔

Reply:
if there's time, I'll flag for refactoring.